app/views/ClaimCreditsForExportsView.scala.html

```diff
@@ -47,14 +48,12 @@
         content = Text(messages("claimCreditsForExports.warning"))
         ))
 
-        <h1 class="govuk-heading-m">@messages("claimCreditsForExports.title")</h1>
         @govukRadios(
             RadiosViewModel.yesNo(
                 field = form("value"),
-                legend = LegendViewModel(messages("claimCreditsForExports.subHeading")),
-                hint = None
-            )
-        )
+                legend = LegendViewModel(messages("claimCreditsForExports.title")).asPageHeading(size = Medium),
```

Comment:
This should not have the asPageHeading modifier as that makes it an h1. The "Exported drinks" above should be the h1. This just needs a withCssClass modifier to add the larger legend size.